build/VSIX.targets

```diff
@@ -0,0 +1,86 @@
+﻿<Project>
```

Comment:
Should we commonize these things? We have similar MSBuild in Razor.

Reply:
maybe one day. For now, we only have this in two places. If using "full" MSBuild becomes a more common requirement for building, we may look into having a switch to run korebuild on full MSBuild from end to end instead